test/testground/network/experiments.go

```diff
@@ -0,0 +1,89 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/test/util/sdkutil"
+	"github.com/cosmos/cosmos-sdk/codec"
+	coretypes "github.com/tendermint/tendermint/types"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+const (
+	UnboundedBlockSize = "unbounded"
+	ConsistentFill     = "consistent-fill"
+)
+
+
+func fillBlocks(ctx context.Context, runenv *runtime.RunEnv, initCtx *run.InitContext, timeout time.Duration) error {
+	seqs := runenv.IntParam(BlobSequencesParam)
+	size := runenv.IntParam(BlobSizesParam)
+	count := runenv.IntParam(BlobsPerSeqParam)
+
+	cmd := NewRunTxSimCommand("txsim-0", timeout, RunTxSimCommandArgs{
+		BlobSequences: seqs,
+		BlobSize:      size,
+		BlobCount:     count,
+	})
+
+	runenv.RecordMessage("leader: sending txsim command")
+
+	_, err := initCtx.SyncClient.Publish(ctx, CommandTopic, cmd)
+	return err
+}
+
+func (l *Leader) unboundedBlockSize(
```

Comment:
Would be great to associate this function with some descriptions.

Reply:
good idea, added here 0140a0a2ec0ee08994a50a2e96142f50c86d2c6c